main.tf

```diff
@@ -418,6 +419,48 @@ data "aws_iam_policy_document" "bucket_policy" {
     }
   }
 
+  dynamic "statement" {
+    for_each = local.enable_cloud_trail_bucket_policy ? [1] : []
+    content {
+      sid = "AWSCloudTrailAclCheck"
+      principals {
+        type        = "Service"
+        identifiers = ["cloudtrail.amazonaws.com"]
+      }
+      actions = [
+        "s3:GetBucketAcl",
+      ]
+      resources = [
+        "arn:aws:s3:::${local.bucket_name}",
```

Comment:
add partition here

Reply:
Not sure what you mean by 'partition' -- just a blank line?

Reply:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/partition

Reply:
instead of hard coding aws, the data partition should be used for govcloud users

Reply:
OK, I was not aware of this -- makes sense.  Will fix.